.cypress/integration/9_integrations.spec.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import {
+  TEST_INTEGRATION_INSTANCE, TEST_SAMPLE_INSTANCE,
+} from '../utils/constants';
+
+let testInstanceSuffix = (Math.random() + 1).toString(36).substring(7);
+let testInstance = `${TEST_INTEGRATION_INSTANCE}_${testInstanceSuffix}`;
+
+const moveToIntegrationsHome = () => {
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/integrations#/available`);
+};
+
+const moveToAvailableNginxIntegration = () => {
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/integrations#/available/nginx`);
+};
+
+const moveToAddedIntegrations = () => {
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/integrations#/installed`);
+};
+
+
+describe('Basic sanity test for integrations plugin', () => {
```

Comment:
Maybe add cleanup on each test.

Reply:
I don't consider this a blocking note.  Create a follow-on issue for tracking.

Reply:
Good call out, sometime I run into issues where i have added too many on my local that it shows up on second page, created an issue: https://github.com/opensearch-project/dashboards-observability/issues/620